app/views/admin/settings/index.html.erb

```diff
@@ -64,12 +64,23 @@
 
       <dt><%= f.label :cache_expiration, t(".fields.cache_expiration") %></dt>
       <dd><%= admin_setting_text_field(f, :cache_expiration, size: "3") %></dd>
+    </dl>
+  </fieldset>
 
+  <fieldset>
+    <legend><%= t(".legend.content") %></legend>
+    <dl>
       <dt><%= admin_setting_checkbox(f, :hide_spam) %></dt>
       <dd><%= f.label :hide_spam, t(".fields.hide_spam") %></dd>
 
       <dt><%= admin_setting_checkbox(f, :guest_comments_off) %></dt>
       <dd><%= f.label :guest_comments_off, t(".fields.guest_comments_off") %></dd>
+
+      <dt><%= f.label :account_age_threshold_for_comment_spam_check, t(".fields.account_age_threshold_for_comment_spam_check") %></dt>
+      <dd>
+        <%= admin_setting_text_field(f, :account_age_threshold_for_comment_spam_check, size: "3", "aria-describedby": "account-age-threshold-for-comment-spam-check-note") %>
+        <p id="account-age-threshold-for-comment-spam-check-note" role="label" class="footnote"><%= t(".fields.account_age_threshold_for_comment_spam_check_note") %></p>
```

Comment:
Could you remove `role="label"` since this isn't acting as a label? (I'm actually not sure label is one of the [allowed values for the role attribute](https://www.w3.org/TR/wai-aria-1.2/#role_definitions). It sounds like something that should be, but it's not on that list or any others I can find.)